plugin/core/sessions.py

```diff
@@ -1713,7 +1717,7 @@ def run_async() -> None:
                 listener = session_view.listener()
                 if not listener:
                     return
-                listener.do_signature_help_async(manual=False)
+                listener.do_signature_help_async(force=True)
```

Comment:
Change the parameter value here but I guess this is really more of an explicit invocation?

Reply:
Not sure if that's what you meant but I've changed it so that you can call `do_signature_help_async` with either "TriggerCharacter" kind AND a char or with other kind and no char. A bit elaborate typing enforces that char is passed when passing TriggerCharacter kind.